packages/error-reporting/README.md

```diff
@@ -44,6 +44,13 @@ var errors = require('@google-cloud/error-reporting')({
 errors.report(new Error('Something broke!'));
 ```
 
+- *One may even catch and report application-wide uncaught errors:*
+
+```js
+var errors = require('@google-cloud/error-reporting')();
+process.on('uncaughtException', e => errors.report(e));
```

Comment:
nit: it would be good to add a `console.error` in the example as well. Most people wouldn't know that installing an uncaught exception handler changes the printing behaviour.